deploy.yaml

```diff
@@ -439,3 +400,66 @@
     - name: Build (and automatically deploy) cachito cachito-worker
       command: oc --namespace "{{ cachito_namespace }}" start-build cachito-worker
       changed_when: true
+
+- name: Set up dockerhub pull secret
```

Comment:
Shouldn't this be done before the other plays? Is the buildroot play the only one that uses the pull secret?

Reply:
It works for everything (definitely) except osbs-registry, where it causes a bizarre problem; the registry (effectively) starts **requiring** the *pull* key for *pushes*...

Reply:
But how come it works even though the pull secret is created *after* all the other builds (cachito, koji etc.) are started? :thinking:

Reply:
Welcome to my world... :imp: 

Actually, @chmeliik, if you have some free time (lol), will you take a look at the cluster at 10.0.79.66? It's in "initial state" (except that IIRC I re-deployed one of the Cachito dCs). Specifically, the failed deploys for koji-*... Ping me for creds.

Reply:
> But how come it works even though the pull secret is created _after_ all the other builds (cachito, koji etc.) are started? thinking

But you have a good point regardless... I'm going to spin up another instance and try changing it.

Reply:
Refactored t0

1. Create dockerhub secret *immediately* after creating projects
2. Remove all traces of dockerhub secret from osbs-registry after we're sure it's deployed successfully

Thanks @chmeliik!